src/resources/PDF-healthHygiene/questions-health-hygiene/question-10/question.html

```diff
@@ -0,0 +1,9 @@
+<p class="question-margin">
```

Comment:
why can't all of these fragments be in one file with div wrapper of it, as it seems very messy with so many mini files.

Reply:
Multi-developer friendly, it has been more reliable than formio backups and has been really easy to work with.